### DISCUSSION TITLE
webssh - ios ssh config

### DISCUSSION THREAD

**Discussion by tngm516** [isontheline/pro.webssh.net#1186] [Q&A]
Regarding this ssh config in ios (on iphone app), is there usage for it? I mean on terminal. you define .ssh/config and you can do "ssh example" but on the iphone it seem useless. if you define on webssh config and add a connection with Host "example" and try to connect "error unable to retrieve ip from dns server" 

Is there way to make this feature work on the iphone app? Or this feature works on webssh app on macbook only?

**Comment by tngm516** [comment]
Or is there specific process to do it in the iphone app to make it work?

**Comment by isontheline** [comment]
Hello @tngm516 👋 

ssh_config is not used inside WebSSH unless you import entries : https://webssh.net/documentation/help/howtos/mashREPL/import/

Could you make it a try?
